content/other/phaseangles/inverse/features/dataset-attributes.md

```diff
@@ -0,0 +1,7 @@
+|Name|Type|Description|
+|-|-|-|
+|angles|dict|Specific angles which apply the polynomial transformation for a specific routine.|
+|epsilons|list|Maximum approximation error $\epsilon$ between the polynomial $P(x)$ and the function $f(x)$ over the chosen interval. Possible values: {0.1, 0.01, 0.001}.|
+|kappas|list|Set of possible values for $\kappa$, which determines the interval for function approximation. Possible values: {1, 5, 50, 100, 250, 500, 1000, 1500}.|
```

Comment:
In case we want to include it, the interval is: [-1, -1/kappa) U (1/kappa, 1]

Reply:
I'm not sure I understand. What is this interval for? The interval where the approximation of 1/x is "good"?

Reply:
I think it's okay not to include it here and keep it in the "Using this Dataset" section